src/containers/Supplier/Components/DeleteComp.tsx

```diff
@@ -1,29 +1,47 @@
 import axios from "axios"
+import { confirmAlert } from 'react-confirm-alert';
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-confirm-alert/src/react-confirm-alert.css';
+import 'react-toastify/dist/ReactToastify.css';
 
 type Props = {
   id: string;
 };
 
 const DeleteComp = ({ id }: Props) => {
   const handleDelete = () => {
-    const confirmDelete = window.confirm('Apakah Anda yakin ingin menghapus data ini?');
-    if (confirmDelete) {
-      axios
-        .delete(`${process.env.API_ENDPOINT}supplier/` + id)
-        .then((response) => {
-          alert(response.data.message)
-          window.location.reload()
-        })
-    }
-  }
+    confirmAlert({
+      title: 'Konfirmasi',
+      message: 'Apakah Anda yakin ingin menghapus data ini?',
+      buttons: [
+        {
+          label: 'Ya',
+          onClick: () => {
+            axios
+            .delete(`${process.env.API_ENDPOINT}supplier/` + id)
+            .then((response) => {
+              toast.success(response.data.message, { autoClose: 1500 });
+              setTimeout(() => {
+                window.location.reload();
+              }, 2000);
```

Comment:
why should reload data? 
just refresh the data.
create context table, contain reload data. when this delete, fetch suppliers with current query

Reply:
done